src/main/java/com/teragrep/lsh_01/config/PayloadConfig.java

```diff
@@ -39,7 +39,7 @@ public PayloadConfig() {
     public void validate() {
         if (splitEnabled) {
             try {
-                Pattern splitPattern = Pattern.compile(splitRegex);
+                Pattern.compile(splitRegex);
```

Comment:
All in all, the return value of the `Pattern.compile(splitRegex)` method is ignored, now its more clean without an assignment.

Might call for a refactoring similar to https://github.com/teragrep/cfe_31/issues/53

Reply:
The refactoring idea is spot on and I didn't come to think of that before. For now I'd say let's keep it like it is because most of the Config files have an empty validate function, so I'd say it's better to have empty validate functions than completely useless objects. I'll make an issue about refactoring the validation process, which in turn would require writing the validations for other configs as well, if they even need validations.